.github/workflows/_meta.yaml

```diff
@@ -0,0 +1,99 @@
+on:
+  workflow_call:
+    inputs:
+      distro:
+        required: true
+        default: 'debian'
+        type: string
+      codenames:
+        description: 'Stringified JSON object listing target distro codenames'
+        required: false
+        default: "[\"bullseye\"]"
+        type: string
+      architectures:
+        description: 'Stringified JSON object listing target architectures'
+        required: false
+        default: "[\"amd64\"]"
+        type: string
+      release:
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  build:
+    name: 'Build FFmpeg'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        release: ${{fromJson(inputs.codenames)}}
+        arch: ${{fromJson(inputs.architectures)}}
+  
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install make and mmv
+        run: sudo apt-get install make mmv
+
+      - name: Build Linux
+        if: ${{ contains(inputs.distro, 'debian') || contains(inputs.distro, 'ubuntu') }}
+        run: ./build ${{ matrix.release }} ${{ matrix.arch }} dist
+
+      - name: Build Windows
+        if: ${{ contains(inputs.distro, 'windows') }}
+        run: ./build-win64 dist
+
+      - name: Upload Linux Artifacts
+        uses: actions/upload-artifact@v2
```

Comment:
you pinned the versions for the other actions but not these and checkout?

Reply:
Pinned those too.